admin/app/dfp/DfpDataCacheJob.scala

```diff
@@ -3,20 +3,17 @@ package dfp
 import common.dfp._
 import common.{ExecutionContexts, Logging}
 import conf.switches.Switches.DfpCachingSwitch
+import dfp.DfpApi.DfpLineItems
 import org.joda.time.DateTime
-import play.api.libs.json.Json
 import play.api.libs.json.Json.{toJson, _}
 import tools.Store
 import scala.concurrent.Future
 
 object DfpDataCacheJob extends ExecutionContexts with Logging {
 
-  case class LineItemLoadSummary(prevCount: Int,
```

Comment:
I've removed most of these fields because they were used for testing and logging. I moved the logging inline, and ensured the tests check the contents of the sequences they manipulate.